content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,1460 @@
+                  type="button" role="tab" aria-controls="dashboard" aria-selected="true">
+            Dashboard
+          </button>
+        </li>
+        <li>
+          <button class="nav-link" data-bs-toggle="tab" id="transactions-tab" data-bs-target="#transactions"
+                  type="button" role="tab" aria-controls="transactions" aria-selected="false">
+            Transactions
+          </button>
+        </li>
+      </ul>
+    </div>
+
+    <div class="divider"></div>
+
+    <div class="main-content tab-content d-flex flex-column flex-shrink-0 p-3">
+
+      <div class="header border-bottom">
+        <h3>
+          Build a XRPL Wallet
+          <small class="text-muted">- Part 6/8</small>
+        </h3>
+      </div>
+
+      <div class="tab-pane fade show active" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab">
+        <h3>Account:</h3>
+        <ul class="list-group">
+          <li class="list-group-item">Classic Address: <strong id="account-address-classic"></strong></li>
+          <li class="list-group-item">X-Address: <strong id="account-address-x"></strong></li>
+          <li class="list-group-item">XRP Balance: <strong id="account-balance"></strong></li>
+          <li class="list-group-item">XRP Reserved: <strong id="account-reserve"></strong></li>
+        </ul>
+        <div class="spacer"></div>
+        <h3>
+          Ledger
+          <small class="text-muted">(Latest validated ledger)</small>
+        </h3>
+        <ul class="list-group">
+          <li class="list-group-item">Ledger Index: <strong id="ledger-index"></strong></li>
+          <li class="list-group-item">Ledger Hash: <strong id="ledger-hash"></strong></li>
+          <li class="list-group-item">Close Time: <strong id="ledger-close-time"></strong></li>
+        </ul>
+      </div>
+
+      <div class="tab-pane fade" id="transactions" role="tabpanel" aria-labelledby="transactions-tab">
+        <h3>Transactions:</h3>
+        <table id="tx-table" class="table">
+          <thead>
+          <tr>
+            <th>Confirmed</th>
+            <th>Type</th>
+            <th>From</th>
+            <th>To</th>
+            <th>Value Delivered</th>
+            <th>Hash</th>
+          </tr>
+          </thead>
+          <tbody></tbody>
+        </table>
+      </div>
+
+    </div>
+
+  </main>
+
+  <dialog id="seed-dialog">
+    <form method="dialog">
+      <div>
+        <label for="seed-input">Enter seed:</label>
+        <input type="text" id="seed-input" name="seed-input" />
+      </div>
+      <div>
+        <button type="submit">Confirm</button>
+      </div>
+    </form>
+  </dialog>
+
+  <dialog id="password-dialog">
+    <form method="dialog">
+      <div>
+        <label for="password-input">Enter password (min-length: 5):</label>
+        <input type="text" id="password-input" name="password-input" />
+      </div>
+      <div>
+        <button type="submit">Confirm</button>
```

Comment:
This is missing the "Change Seed" option used in the example code `6-styling/view/template.html` (Be careful copying directly because the relative imports are different for bootstrap and such)

Reply:
Done